internal/services/eventhub/eventhub_consumer_group_resource.go

```diff
@@ -208,3 +210,10 @@ func (r ConsumerGroupResource) ModelObject() interface{} {
 func (r ConsumerGroupResource) IDValidationFunc() pluginsdk.SchemaValidateFunc {
 	return consumergroups.ValidateConsumerGroupID
 }
+
+func (r ConsumerGroupResource) StateUpgradeDataFunc() sdk.StateUpgradeData {
+	return sdk.StateUpgradeData{
+		SchemaVersion: 1,
+		Upgraders:     map[int]pluginsdk.StateUpgrade{0: migration.ConsumerGroupsV0ToV1{}},
```

Comment:
can we make this:

```suggestion
		Upgraders:     map[int]pluginsdk.StateUpgrade{
			0: migration.ConsumerGroupsV0ToV1{},
		},
```

(not a biggie, it's just easier to read)